### PR TITLE
NeuralNet test

### PR DESCRIPTION
**Pull request by m2kulkarni**
Added test to make sure the logits from the Torch and C implementations are within tolerance.

- Modified puffer_drive_weights.bin to a different file that matches the corresponding .pt file
- added another workflow
- `test_forward` method in bindings that returns `action, logit` pair

### CODE REVIEW

**Review by eugenevinitsky — APPROVED** [review]
This seems great to me, thanks @m2kulkarni!
